packages/amplify-migration-tests/src/__tests__/migration_tests_v10/__snapshots__/storage.migration.test.ts.snap

```diff
@@ -0,0 +1,24 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`storage category migration from v10 to latest ...pull and push should not drift with new amplify version 1`] = `
+"Resources
+[~] AWS::Lambda::Function UserPoolClientLambda 
+ └─ [~] Runtime
+     ├─ [-] nodejs14.x
+     └─ [+] nodejs16.x
+
+"
+`;
+
+exports[`storage category migration from v10 to latest ...pull and push should not drift with new amplify version 2`] = `
+"Outputs
+[+] Output LambdaExecutionRoleArn: {\\"Value\\":{\\"Fn::GetAtt\\":[\\"LambdaExecutionRole\\",\\"Arn\\"]}}
+
+Resources
+[~] AWS::Lambda::Function UserPoolClientLambda 
+ └─ [~] Runtime
+     ├─ [-] nodejs14.x
+     └─ [+] nodejs16.x
+
+"
+`;
```

Comment:
is this the output from the CFN diff comparison? 

does this test suggest that these differences are valid (ie, node14 and node16 difference is known , and OK)

And that LamdbaExecutionRoleArn is new in v11?

Reply:
Yes, these are outputs from CFN diff comparison.

The approach I took originally was to assert that cfn diff is empty, which worked well for single ddb. But as I progressed in adding more resources to the mix I realized that this goal is not reasonable, because there are and will be changes that are expected to happen. 
Therefore I changed strategy to what's present in this PR, i.e.:
- dump cfn diff output to snapshot
- inspect diff and check if changes are expected (like nodejs runtime bump) OR not breaking (like that extra output).

Than having snapshot as baseline any new change will show up in PR diff and both author and reviewer can assess if new change is acceptable.